service/bfg/bfg.go

```diff
@@ -1427,6 +1429,37 @@ func (s *Server) Run(pctx context.Context) error {
 		publicHttpErrCh <- publicHttpServer.ListenAndServe()
 	}()
 
+	if s.cfg.PprofListenAddress != "" {
+		pprofMux := http.NewServeMux()
```

Comment:
hey @joshuasing , I am sure you thought about this, but if I am not mistaken, this code is repeated across the services, could this code be importable and callable instead? as to not repeat the logic?

Reply:
Yes, it could be and I would like to (there's a note in the pr message)

I was unsure of where to put a package for this, and what to name it.

Reply:
ah I missed the note 🤦 I think we should put this in a package.  what to name it, I am unsure.  perhaps `profile`?